record/record.go

```diff
@@ -24,11 +24,12 @@
 //		records := record.NewReader(r)
 //		for {
 //			rec, err := records.Next()
-//			if err == io.EOF {
+//			if err == io.EOF || err == io.ErrUnexpectedEOF {
 //				break
 //			}
 //			if err != nil {
-//				return nil, err
+//				r.Recover()  // Skip corrupted records.
```

Comment:
Recover returns an error, which you're ignoring here. Should we change the example code or the API?